guidelines/creating_dockerfiles.md

```diff
@@ -0,0 +1,71 @@
+# Guidelines for creating Docker images for Kubeflow applications
+
+Authors: duselvar@cisco.com, jlewi@google.com, adselvaraj@cisco, jtf.github@gmail.com
+
+## Objective
+
+The purpose of this document is to outline a process for creating docker images maintained by our community for Kubeflow Applications.
```

Comment:
lower case the "A" in "Application" (in other places too)?

Reply:
I'll change the caps to small.